app/components/community_evidence_submission_modal_component.rb

```diff
@@ -0,0 +1,43 @@
+class CommunityEvidenceSubmissionModalComponent < ViewComponent::Base
+  def initialize(activity:, achievement: nil, button_class: nil)
+    @activity = activity
+    @achievement = achievement
+    @button_class = button_class
+  end
+
+  def achievement_complete?
```

Comment:
I wonder if we could move this method into the achievement model, as we have it in two different components?

Call it `community_achievement_complete?` or something like that?